tests/overlapping-segments-after-rounding.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh -e
+
+PATCHELF=$(readlink -f "../src/patchelf")
+SCRATCH="scratch/$(basename "$0" .sh)"
+READELF=${READELF:-readelf}
+
+EXEC_NAME="overlapping-segments-after-rounding"
+
+if test "$(uname -i)" = x86_64; then
+    rm -rf "${SCRATCH}"
+    mkdir -p "${SCRATCH}"
+
+    cp "${srcdir}/${EXEC_NAME}.xz" ${SCRATCH}/
```

Comment:
Can you skip the test if `xz` is not present on a system. I will make sure it will be available in CI.

Reply:
Yes I think so. Although, I should be present, as it is passing on the CI on my side: https://github.com/brenoguim/patchelf/tree/breno.446

The file is not really big though. We can even live with having it as a whole in the repo. I think it was 200k after unxz. Will double check.

Reply:
Yes. `stdenv` we have xz but I see many distributors and other users doing `make check` during installation.